packages/firebase_ml_vision/example/test_driver/firebase_ml_vision.dart

```diff
@@ -62,6 +62,21 @@ void main() {
       });
     });
 
+    group('$VisionEdgeImageLabeler', () {
```

Comment:
Could you separate this to another file when you pull the latest changes in?

Reply:
will do